tests/operon-runtime/runtime.test.ts

```diff
@@ -104,7 +163,7 @@ database:
   system_database: 'hello_systemdb'
   connectionTimeoutMillis: 3000
   user_dbclient: 'knex'
-localRuntimeConfig:
+runtimeConfig:
```

Comment:
```suggestion
  runtimeConfig:
```

Reply:
I'd have to test, but I'm pretty sure pushing `runtimeConfig` down a level would break the code.

Reply:
As per slack chat w/ @maxdml, we don't want to do this